src/ft2font_wrapper.cpp

```diff
@@ -1431,6 +1441,34 @@ PyFT2Font_get_image(PyFT2Font *self)
     return py::array_t<unsigned char>(dims, im.get_buffer());
 }
 
+const char *PyFT2Font__get_type1_encoding_vector__doc__ = R"""(
+    Return a list mapping CharString indices of a Type 1 font to FreeType glyph indices.
+
+    Returns
+    -------
+    list[int]
+)""";
+
+static std::array<FT_UInt, 256>
+PyFT2Font__get_type1_encoding_vector(PyFT2Font *self)
+{
+    auto face = self->x->get_face();
+    auto indices = std::array<FT_UInt, 256>{};
+    for (auto i = 0; i < indices.size(); ++i) {
+        auto len = FT_Get_PS_Font_Value(face, PS_DICT_ENCODING_ENTRY, i, nullptr, 0);
+        if (len == -1) {
+            throw std::runtime_error{
+                "FT_Get_PS_Font_Value tried to access non-existent value "
+                "PS_DICT_ENCODING_ENTRY #" + std::to_string(i) + " of "
+                + PyFT2Font_fname(self).cast<std::string>()};
+        }
+        auto buf = std::unique_ptr<char[]>{new char[len]};
```

Comment:
I think you can shorten this with `std::make_unique`?
```suggestion
        auto buf = std::make_unique<char[]>(len);
```

Reply:
oooh cute, I didn't know that.